src/Compiler/Checking/CheckFormatStrings.fs

```diff
@@ -371,7 +371,7 @@ let parseFormatStringInternal
             // type checker.  They should always have '(...)' after for format string.
             let requireAndSkipInterpolationHoleFormat i =
                 if i < len && fmt[i] = '(' then
-                    let i2 = fmt.IndexOfOrdinal(")", i+1)
+                    let i2 = fmt.IndexOf(')', i+1)
```

Comment:
Why not ordinal here?

Reply:
char overloads are always Ordinal. My PR is larger than https://github.com/dotnet/fsharp/pull/16439 (that introduced `IndexOfOrdinal` ext method) and established usage of char overload.
\+ potentially it's faster on netfx (on Core it delegated to the same Span function)

Reply:
Also there is https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1865-ca1867 and https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1847